backend/src/index.js

```diff
@@ -3,18 +3,18 @@ import bodyParser from "body-parser";
 import circuitRouter from "./routes/circuit-router";
 import userRouter from "./routes/user-router";
 import authDemoRouter from "./routes/auth-demo-router";
-import cors from "cors"
+import cors from "cors";
```

Comment:
Can you run `npm run lint` in the repo so fix up all the small issues like this.